src/smpmap.cpp

```diff
@@ -88,9 +88,9 @@ void ChunkColumn::update(std::uint8_t sec_coord,
     const std::vector<std::int64_t>& records) {
   auto& section = sections[sec_coord];
   for(auto el : records) {
-    std::uint8_t y = el&0xF;
-    std::uint8_t z = (el>>=4)&0xF;
-    std::uint8_t x = (el>>=4)&0xF;
+    std::int64_t z = el&0xF;
+    std::int64_t y = (el>>=4)&0xF;
+    std::int64_t x = (el>>=4)&0xF;
```

Comment:
Were these broken? The max value of chunk relative coordinates is 255

Reply:
if I left them as they were, they did not have valid values, this was a quick fix that solved it